src/main/resources/application.yml.example

```diff
@@ -3,13 +3,25 @@ spring:
     web-environment: false
 
 rebazer:
-  team: your_company
-  user: service_user
-  pass: dont_use_this_pass_at_home
-  workspace: ./rebazer-workspace
-  garbageCollectionCountdown: 20
-  repos:
-    - name: the_repo_name
-      branch: master
-    - name: the_other_repo_name
-      branch: master
+  workspace: ./rebazer-workspace # optional
+  garbageCollectionCountdown: 20 # optional
+  pollInterval: 60 # optional
+  hosts:
+    - type: BITBUCKET # possible values GITHUB or BITBUCKET
```

Comment:
I know that uppercase is necessary to match the corresponding enum. But it seems quite out of place here to have upper case... Any preferences/solutions?

Reply:
I thought about that, but writing the enum values lower case is worse